API.md

```diff
@@ -252,7 +252,7 @@ const ManagedCompactTableAccordionGroup = manageState(CompactTableAccordionGroup
 Prop | Required | Default | Type | Description
 :--- | :------- | :------ | :--- | :----------
  `changeOnTitleClick` |  | ```false``` | bool | 
- `children` | true | `````` | node | 
+ `children` |  | ```undefined``` | node | 
```

Comment:
odd that this has changed - altho for this prop i guess that it shouldn't have been `required`

Reply:
It is odd that this has changed but I think we're prob okay to leave as-is for now.